mypy/plugins/enums.py

```diff
@@ -0,0 +1,69 @@
+from typing import Optional
```

Comment:
Add docstring explaining the purpose of the plugin and the implementation approach (how this uses `last_known_value` at least).